modules/videoio/src/cap_msmf.cpp

```diff
@@ -44,7 +44,7 @@
    Media Foundation-based Video Capturing module is based on
    videoInput library by Evgeny Pereguda:
    http://www.codeproject.com/Articles/559437/Capturing-of-video-from-web-camera-on-Windows-7-an
-   Originaly licensed under The Code Project Open License (CPOL) 1.02:
+   Originally licensed under The Code Project Open License (CPOL) 1.02:
```

Comment:
Is this OK to correct even though it's in a license block?

Reply:
This is a typo fix, so it is OK.